packages/core/admin/server/controllers/api-token.js

```diff
@@ -41,4 +41,12 @@ module.exports = {
 
     ctx.send({ data: apiTokens });
   },
+
+  async revoke(ctx) {
```

Comment:
Any reason to call this revoke ? I looks to me it doesn't convey the actual functionally.